pkg/config/config.go

```diff
@@ -26,6 +53,8 @@ type Install struct {
 
 type Config struct {
 	Install *Install `yaml:"install,omitempty"`
+	Cluster Cluster  `yaml:"cluster,omitempty"`
```

Comment:
I do wonder if it does belong here, providers can extend the config without attaching to the main config structure

Reply:
We could move Cluster to the sdk and leave it to the plugin to parse.  Given C3OS and the agent don't interact with the cluster configuration I think this makes sense.